test/genall5.c

```diff
@@ -2755,12 +2755,13 @@ tend_zoo(hid_t fid, const char *base_path, struct timespec *lastmsgtime,
     }
 out:
     if (!ok) {
```

Comment:
The original code set the `lastmsgtime` to 0.  `below_speed_limit` would always return true.  I changed it to use the `lastmsgtime` passed into this function, which is set by the caller.

Reply:
This function is no longer used by the zoo test after refactoring the code.  So I will close this comment.